4.x/typescript/Recenter/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Recenter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/arcgis-js-api": "^4.18.0"
```

Comment:
It looks like the @types/arcgis-js-api also need to be bumped to `^4.19.0`

Reply:
Ahhh I didn't save that. I had it on my machine and after I pushed that change I still had one unsaved edit, this was that. Thanks :-) I'll fix it now.

Reply:
I missed one more item. `"license": "Apache-2.0"`.

Reply:
@andygup could you doublecheck on that version? It is showing correctly on my end.

https://github.com/Esri/jsapi-resources/blob/549e0f4c626a7fe87ca216bd5ff8dd6078014cf9/4.x/typescript/Recenter/package.json

Reply:
TS looks good, I'm seeing `"@types/arcgis-js-api": "^4.19.0"`

Reply:
@andygup I'll fix the license. Just saw your comment.